build/docker-compose.yml

```diff
@@ -19,3 +19,8 @@ services:
     volumes:
       - "/tmp/localstack:/tmp/localstack"
       - "/var/run/docker.sock:/var/run/docker.sock"
+
+  kafka:
+    image: bashj79/kafka-kraft
```

Comment:
What do you think about switching to [this](https://github.com/wurstmeister/kafka-docker) repository?
It's a pretty up-to-date and maintained Kafka docker image.

Reply:
The reason I chose `bashj79/kafka-kraft` because this one works without Zookeeper dependency for Kafka. Confluent guys already removed zookeeper in Kafka 2.8 and moved to store kafka metadata inside internal kafka topics using Raft for consensus.

For local development this will be lighter.

Reply:
Understood! Makes sense.